### ISSUE TITLE
Overwriting of Existing Resources

### ISSUE THREAD

**Issue by rmcolbert** [vmware/terraform-provider-avi#358]
### Describe the bug

The Terraform provider does not throw an error if it is being asked to create a resource that already exists.

For example, if you manually create a health monitor named "TEST123" via GUI, REST or CLI, then use Terraform to create an avi_healthmonitor resource using the same name, Terraform will overwrite the existing TEST123 monitor.

This is in direct conflict to all other Terraform providers which prevent managing existing resources unless they are imported into the TF state fire. Furthermore, it exposes a risk to the infrastructure where multiple development teams could mangle/cross-workspace manage the same objects.

### Reproduction steps

```bash
1. Create a Ping monitor via the GUI with unusual values (description, high test values etc)
2. Create avi_healthmonitor terraform resource using the same monitor name and type but different values
3. Run terraform apply
4. Confirm only 1 Ping monitor exists with the specified name and that all values have been changed to the Terraform inputs
```

### Expected behavior

Terraform should throw an exception that the resource already exists and that it needs to be imported in to the state file to be managed.

### Additional context

I have confirmed this behavior on Monitors and GSLB services but suspect it exists in many other resources.

**Comment by chandanapatnala** [comment]
" terraform refresh " command can be used to update the state file. An error isn't thrown that the resource already exists but however it shows a diff that an existing resource is being updated.

**Comment by rmcolbert** [comment]
> " terraform refresh " command can be used to update the state file. An error isn't thrown that the resource already exists but however it shows a diff that an existing resource is being updated. 

That is not how Terraform is supposed to work. If a resource already exists, it should throw a conflict error like every other Terraform provider. 

If Terraform hasn't brought the resource under management yet, a state refresh isn't going to do anything because Terraform itself doesn't know to query that object.

**Comment by Rohan-sss1** [comment]
HI @rmcolbert this is fixed, checkout the provider version 22.1.6
